dev/import-beats/streams_config_parser.go

```diff
@@ -0,0 +1,374 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"text/template/parse"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/package-registry/util"
+)
+
+type streamConfigParsed struct {
+	tree *parse.Tree
+}
+
+func parseStreamConfig(content []byte) (*streamConfigParsed, error) {
+	mapOfParsed, err := parse.Parse("hello", string(content), "", "", map[string]interface{}{
```

Comment:
"hello"?

Reply:
There is only single template, so need to define a specific template name hence this "hello world". I can rename it to something meaningful

Reply:
ah, got it. Something more descriptive would be nice.

Reply:
Fixed.